lib/util/glob.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @fileoverview An inherited `glob.GlobSync` to support .gitignore patterns.
+ * @author Kael Zhang
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var Sync = require("glob").GlobSync,
+    path = require("path"),
+    util = require("util");
+
+//------------------------------------------------------------------------------
+// Private
+//------------------------------------------------------------------------------
+
+var IGNORE = typeof Symbol === "function" ? Symbol("ignore") : "_shouldIgnore";
```

Comment:
Node.js supports symbols since 0.12. ESLint requires `>=Node 4`, so we can use symbols always.

> http://node.green/#Symbol